pkg/ddevapp/backdrop.go

```diff
@@ -0,0 +1,143 @@
+package ddevapp
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/Masterminds/sprig"
+	"github.com/drud/ddev/pkg/appports"
+	"github.com/drud/ddev/pkg/output"
+	"github.com/drud/ddev/pkg/util"
+)
+
+type BackdropSettings struct {
```

Comment:
These are the same as DrupalSettings, true? Would anybody know or care if we just used the Drupal struct?

Reply:
DrupalSettings is a superset of BackdropSettings, but it's only one element, I think.

Reply:
I kind of think that that we've over-engineered this whole thing based on expectations from a long, long time ago. In the meantime, we've gone with standard members "db/db/db/db", and there's never a reason to use a table prefix, etc. So we may want to calm down and simplify the whole thing. OTOH, it may be that we'll discover something new in typo3, which is a bit different from what we've done in the past.